core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -865,6 +866,31 @@ public void testRelTypePropertiesIncludes() {
         } );
     }
 
+    @Test
+    public void testRelTypePropertiesIncludesWithDoubleRel() {
+        db.executeTransactionally( "CREATE (a:A)-[:FIRST {a: 1}]->(b:B), (a)-[:SECOND {b: '2'}]->(c)" );
```

Comment:
This doesn't look like the relationships type with an underscore gets tested.....should we have a test with something like below for the query?

`CREATE (a:A)-[:FIRST_A {a: 1}]->(b:B), (a)-[:SECOND_B {b: '2'}]->(c)`

Reply:
Yes, the issue title can be a little bit confusing. Indeed, the pr fix it does not depend on underscores but on multiple rel nodes.
But, to be sure, I updated the test with underscores.

However, as I answered in the issue here https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/2383#issuecomment-1049737242, 
maybe the  issuecause is another (or both of them),
and in that case is solvable without code edit.

Anyhow, this pr should still be valid because I think it's a bug